### ISSUE TITLE
Check all benchmark results

### ISSUE THREAD

**Issue by athei** [paritytech/polkadot-sdk#6722]
Before releasing on any value bearing chain we need to make sure that all benchmark results and hence weights do make sense.

**Comment by mordamax** [comment]
@athei related / similar to https://github.com/paritytech/polkadot-sdk/issues/6196 ?

**Comment by athei** [comment]
This is more than just running the benchmarks. We did a lot of changes to the pallet without paying much attention to the benchmarks. I need to go through them and make sure they make sense. Some of them need to be reworked like https://github.com/paritytech/polkadot-sdk/issues/6157

**Comment by mordamax** [comment]
@athei would https://github.com/paritytech/polkadot-sdk/issues/6196 bring any problems or should be ok?

**Comment by athei** [comment]
Thats okay. Just re-running the benchmarks should not break anything. Production chains will have their own benchmarks anyways.

**Comment by kianenigma** [comment]
#5792 related issues about generally having correct weights across the board. 

Parachains should generally have "correct" weights at all times.

**Comment by athei** [comment]
Yeah the weights are generally correct. Just added this as a mental reminder to go through them manually when adding the pallet for the first time to make sure nothing goes south.
